packages/mui-base/src/TextareaAutosize/TextareaAutoSize.test.tsx

```diff
@@ -13,29 +13,38 @@ import TextareaAutosize from '@mui/base/TextareaAutosize';
 describe('<TextareaAutosize />', () => {
   const { clock, render } = createRenderer();
 
-  describeConformance(<TextareaAutosize />, () => ({
-    inheritComponent: 'textarea',
-    refInstanceof: window.HTMLTextAreaElement,
-    skip: [
-      'rootClass',
-      'componentProp',
-      'componentsProp',
-      'themeDefaultProps',
-      'themeStyleOverrides',
-      'themeVariants',
-    ],
-  }));
+  describeConformance(
+    <TextareaAutosize minRows={1} />,
+    () =>
+      ({
+        inheritComponent: 'textarea',
+        refInstanceof: window.HTMLTextAreaElement,
+
+        skip: [
+          'rootClass',
+          'componentProp',
+          'componentsProp',
+          'themeDefaultProps',
+          'themeStyleOverrides',
+          'themeVariants',
+        ],
+      } as any),
```

Comment:
Casting to `any` should be used as a last resort. In this case, it should be possible to correct the code so it's not needed.